src/inverseRobot.js

```diff
@@ -7,7 +7,15 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const entries = Object.entries(robot).map(([key, value]) => [value, key]);
+  const allElements = entries.flatMap(arr => [...arr]);
+  const filteredArray = [...new Set(allElements)];
```

Comment:
```suggestion
  const values = Object.values(robot);
  const filteredArray = [...new Set(values)];
```

you can simplify your solution